src/main/java/org/jabref/logic/integrity/EditionChecker.java

```diff
@@ -44,10 +45,15 @@ public EditionChecker(BibDatabaseContext bibDatabaseContext) {
         }
 
         //BibTeX
-        if (!bibDatabaseContextEdition.isBiblatexMode() && !FIRST_LETTER_CAPITALIZED.test(value.trim())) {
+        if (!bibDatabaseContextEdition.isBiblatexMode() && !FIRST_LETTER_CAPITALIZED.test(value.trim()) &&
+                !value.equals(FIRST_EDITION)) {
             return Optional.of(Localization.lang("should have the first letter capitalized"));
         }
 
+        if (value.equals(FIRST_EDITION)) {
```

Comment:
Personally, I would move this check before the biblatex/bibtex checks (then you don't need to modify the conditions of these if statements).

Reply:
@tobiasdiez yes, it makes more sense. Updated PR.